config/src/main/java/org/axonframework/config/AggregateConfigurer.java

```diff
@@ -84,6 +88,9 @@ protected AggregateConfigurer(Class<A> aggregate) {
                                                     c -> NoSnapshotTriggerDefinition.INSTANCE);
         aggregateFactory =
                 new Component<>(() -> parent, name("aggregateFactory"), c -> new GenericAggregateFactory<>(aggregate));
+        filter = new Component<>(() -> parent, "filter<" + aggregate.getSimpleName() + ">", c -> null);
```

Comment:
Shouldn't this field be named `eventFilter`, thus mirroring the configuration method?

Reply:
I'd state the same name change for the `Component#name` field.